### PR TITLE
add missing various document comments

### PR DESCRIPTION
**Pull request by rasanga**
Document comments are missing on various classes

### CODE REVIEW

File: src/AppBundle/EventListener/ControllerListener.php [hunk 1]
```diff
@@ -28,13 +28,22 @@
  */
 class ControllerListener
 {
+    /**
+     * @var SourceCodeExtension
+     */
     protected $twigExtension;
 
+    /**
+     * @param SourceCodeExtension $twigExtension
```
Comment (stof) [inline src/AppBundle/EventListener/ControllerListener.php]: this adds exactly no value compared to what is provided by the method signature. In such case, we don't add meaningless phpdoc in Symfony

File: src/AppBundle/Twig/SourceCodeExtension.php [hunk 2]
```diff
@@ -58,6 +68,9 @@ public function showSourceCode(\Twig_Environment $twig, $template)
         ));
     }
 
+    /**
+     * @return array|void
```
Comment (stof) [inline src/AppBundle/Twig/SourceCodeExtension.php]: null, not void

**Comment by stof** [comment]
Your PR cannot be merged, because you worked on an outdated codebase. Please rebase it to fix conflicts

File: src/AppBundle/Twig/SourceCodeExtension.php (hunk 2 above)
Comment (rasanga) [inline src/AppBundle/Twig/SourceCodeExtension.php]: Good spot, I will correct that

File: src/AppBundle/EventListener/ControllerListener.php (hunk 1 above)
Comment (rasanga) [inline src/AppBundle/EventListener/ControllerListener.php]: Yes, I agree

**Comment by rasanga** [comment]
Fixed all issues mentioned in code review.

File: src/AppBundle/EventListener/ControllerListener.php [hunk 3]
```diff
@@ -35,6 +38,9 @@ public function __construct(SourceCodeExtension $twigExtension)
         $this->twigExtension = $twigExtension;
     }
 
+    /**
+     * @param FilterControllerEvent $event
+     */
```
Comment (xabbuh) [inline src/AppBundle/EventListener/ControllerListener.php]: This docblock doesn't add any value (the parameter type can be derived from the type hint).

File: src/AppBundle/Twig/AppExtension.php [hunk 4]
```diff
@@ -26,8 +26,14 @@
...
 class AppExtension extends \Twig_Extension
 {
+    /**
+     * @var Markdown
+     */
     private $parser;
 
+    /**
+     * @param Markdown $parser
+     */
```
Comment (xabbuh) [inline src/AppBundle/Twig/AppExtension.php]: same here

**Comment by javiereguiluz** [comment]
@rasanga thanks for adding all the missing PHPdocs and for your patience during the review process!

**Pull request closed without merging**